CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
```

Comment:
also https://github.com/microsoftgraph/msgraph-sdk-go-core/blob/main/version.go needs to be updated